tests/unit/test_llm_utils.py

```diff
@@ -0,0 +1,44 @@
+import inspect
+import pyspark.sql.functions as F
+
+from databricks.labs.dqx.check_funcs import make_condition, register_rule
+from databricks.labs.dqx.llm.utils import get_check_function_definition
+
+
+@register_rule("row")
+def dummy_custom_check_function_test(column: str, suffix: str):
+    """
+    Test the custom check function.
+    """
+    return make_condition(
+        F.col(column).endswith(suffix), f"Column {column} ends with {suffix}", f"{column}_ends_with_{suffix}"
+    )
+
+
+def test_get_check_function_definition():
+    custom_check_functions = {"dummy_custom_check_function_test": dummy_custom_check_function_test}
+    result = list(
+        filter(
+            lambda x: x['name'] == 'dummy_custom_check_function_test',
```

Comment:
`test_get_check_function_definition_with_missing_custom_check_functions` coverts the case when no `custom_check_functions ` is provided. Please also add case when the function is not defined in `custom_check_functions` but the `custom_check_functions` is passed.